meetings/2023-12-13.md

```diff
@@ -0,0 +1,127 @@
+* eP: Thank you for the work you do. Often people who work on specs/implementing become very focused on that and sometimes not good outreach, so what you do is important. How big is the audience of wedistribute? And do you work with articles or other media (video, interviews)? 
+* LH: It's not big. We're the largest probably on the fediverse but ??? Mainly articles, there's work on a podcast, and interviews. Even developers don't talk to each so it's a way of getting to know other similar projects. 
+* DO: No set number. We had an article last week about 1000 reads. Depends on the topic. We have accounts on decentralized and centralized platforms. Making an effort to bolster our social media presence. We have started a podcast that we haven't publicly lauched but we have 4-5 episodes with developers, co-authors of AP and some with AT protocol in the works.. we want to extend that to cover Solid as well. We have an informal relationship with Sebastien Rosset. We're also working on some video media utilising PeerTube platform. Publication has been around for some time. 
+* SC: FYI, there was some coverage of dweb/social approaches in https://gitlab.com/bluesky-community1/decentralized-ecosystem that you might find useful, if not already in your radar.
+* SC: Happy to help if you'd like to go over Solid Protocol or other specs: https://solidproject.org/TR/
+
+
+### WIP Implementation Feedback
+
+* SC: We'll allocate some time for implementation feedback or interest to implement. Links to products/projects and demos welcome.
+* eP: More of a question: been working on establishing mutual relationship between two social agents. Have a working flow based on SAI. Currently relies on SAI sending through a message (sms, email). I want to optimize copy/paste flow so started using [webshare API](https://www.w3.org/TR/web-share/). Wanted to ask if people have experience with [web share target](https://w3c.github.io/web-share-target/) because it's not consistence across platforms. 
+* VB: May want to ask this in solid/app-development chat.
+
+
+### Solid CG Chair Election Procedure
+URL: https://github.com/solid/specification/discussions/582
+
+* SC: Voting closed. Requested W3C Team (Community Process) to give us the anonymised ballot data that's to be computed: https://lists.w3.org/Archives/Public/public-solid/2023Dec/0016.html
+* eP: W3C takes holiday break next week?
+* SC: Yes, I think 18th. Officially. 
+* eP: If they find it inconvenient to make a PR and send you data instead that'd be okay with you making the PR and they just approve or comment on that PR. I don't want to delay because they don't feel comfortable. 
+* SC: They're aware of the procedure. If there were issues they'd have mentioned or would let us know or just send me the data anyway. I don't expect any slowdown. They give us the data, we compute, we give to them, they approve and announce it. 
```

Comment:
```suggestion
* eP: If they find it inconvenient to make a PR and send you data instead, that'd be okay, with you making the PR and they just approve or comment on that PR. I don't want to delay because they don't feel comfortable. 
* SC: They're aware of the procedure. If there were issues, they'd have mentioned or would let us know or just send me the data anyway. I don't expect any slowdown. They give us the data, we compute, we give to them, they approve and announce it. 
```